src/Symfony/Component/VarExporter/LazyObjectInterface.php

```diff
@@ -16,7 +16,7 @@ interface LazyObjectInterface
     /**
      * Returns whether the object is initialized.
```

Comment:
The behavior of the argument should be described IMO

Reply:
now that the argument is a simple boolean, it might be self explanatiry.